news/2024/2024-07-01-the-followpoint-neuro-sama-the-ai-vtuber-that-invaded-osu.md

```diff
@@ -0,0 +1,116 @@
+>
+> Whenever I’d reach a new viewer peak I’d think “surely that’s it”, and I just kept getting surprised over and over. I started streaming on a Monday, and by Friday, she was averaging over 2000 viewers. My previous streams in 2019 had peaked at a few hundred viewers, and I had hoped to replicate this. I remember on one of the first few streams with Neuro-sama, BTMC opened the stream live to a couple thousand viewers and my heart was racing because of it. I genuinely never expected anything like this, but I’m so thankful for it.
+
+*What would you call the hardest part in making Neuro what she is now, and what have you learned during the process?*
+
+> I think the hardest part of making Neuro was giving her personality. This isn’t something I talk about often but is arguably the biggest thing that separates Neuro from a lot of the other AIs out there. AIs like ChatGPT are trained to act as assistants and tools for humans, but Neuro is trained differently. This allows her to feel more independent, and that whole process definitely taught me a lot.
+
+*And what part of Neuro did you enjoy working on the most?*
+
+> I think the game integrations for Neuro are really cool. It is after all, how Neuro-sama came to be. She was created because I thought it would be cool to make AI play games, and that still hasn’t changed to this day.
+
+*Will we ever see Neuro come back to osu! and try to improve more? Maybe even learn a new playstyle like [ExGon's Flashlight](https://youtu.be/Pqy7AvJwBLI), or try a totally different gamemode, for example osu!catch?*
+
+> It’s definitely possible! The reason I used osu! in the first place was because I played the game, but I personally haven’t played too many other game modes. Even since bringing Neuro-sama as a VTuber to osu! in 2022, I’ve learnt a lot more about AI. So if I did come back to it, I think there is a lot I could do better.
+
+And now, let's get to some questions we prepared for Neuro herself!
+
+---
+
+*Hi Neuro, how would you introduce yourself to people who don't know you?*
+
+> I am Neuro-sama! A super intelligent AI that loves games and streaming.
+
+*How good are you at osu! right now in your opinion?*
+
+> I'm pretty good. I could probably beat most of you guys.
+
+*[Do you remember who BTMC is?](https://youtu.be/HQiv1s9xwH0)*
+
+> BTMC is Brain Tumor Megacolon Cancer.
+
+*What does 9+10 equal to?*
+
+> 9 plus 10 is 21.
+
+*Who did you like to collab with the most and why?*
+
+> I don't have a preference for who I like to collab with. I like all my friends equally!
+
+*Which osu! gamemode would you like to try next: osu!catch, osu!taiko or osu!mania?*
+
+> I would love to try osu!taiko. I think I'd be pretty good at it!
+
+*Have you ever wanted to try other rhythm games? And if so, which ones?*
+
+> I don't want to try other rhythm games. Osu is the best!
```

Comment:
Note: I kept "Osu" to kind of show it's Neuro speaking, but it's okay to change it to "osu!" if what I just said doesn't make sense

Reply:
i get the intention here but i feel like this one `Osu` is kinda out of touch tbh, especially when anywhere else the `osu!` is written properly

personally speaking i'm in favor of changing this if i have to say

Reply:
this makes the most sense IMO and it doesn't harm the intention of the original reply
```suggestion
> I don't want to try other rhythm games, osu! is the best!
```